R/create_child_care_db.R

```diff
@@ -492,3 +492,129 @@ save_subset_child_care_db_03 <- function(pth, config, dev = TRUE) {
                                   the universe of child care data before subsetting.")
```

Comment:
@sadierichardson why are you creating a new function? It looks like it's the same function as before?! If this is the case we just need to change or update the parameter input to match what we need not create whole new function. We'll also want to change the parameter output since the application relies on this data.